app.json

```diff
@@ -95,6 +95,10 @@
       "description": null,
       "required": false
     },
+    "MAX_S3_GET_ITERATIONS": {
+      "description": null,
```

Comment:
Could you give this a description? We'll need them anyway once we upgrade to the newer config stuff